### DISCUSSION TITLE
Suggestions for more games to add (experimental first)

### DISCUSSION THREAD

**Discussion by MesaBlack** [magcius/noclip.website#662] [General]
- [ ] Half-Life: Opposing Force
- [ ] Half-Life: Blue Shift
- [ ] Half-Life: Decay
- [ ] FNAF: Security Breach
- [ ] Portal: Still Alive
- [ ] The ability to load Goldsrc/Source/Xash3d mods' maps
- [ ] The 3D Prince of Persia games
- [ ] Pepsiman

**Comment by magcius** [comment]
I do not take suggestions for games. If people want to contribute games, I am happy to mentor others on the codebase. This is your third suggestion where I've replied something along these lines. Please do not open more.
